### DISCUSSION TITLE
How to disable the emission of traces for requests to the healthcheck endpoint?

### DISCUSSION THREAD

**Discussion by Kralizek** [microsoft/aspire#4198] [Q&A]
My Application Insight is mostly filled with healthcheck requests. I'd love to get rid of them to give more visibility to the actual load.

I'd expect this to be part of the ServiceDefaults project.

**Comment by MattParkerDev** [comment]
@davidfowl I think this would be useful

Add an ASP.NET Core API, and registering the health checks with Aspire:
```
var api = builder.AddProject<WebApi>("api")
	.WithHttpsHealthCheck("/health")
	.WithHttpsHealthCheck("/alive");
```

Means they show up on the resource here:

![image](https://github.com/user-attachments/assets/67efb35e-2b64-4240-843e-858f34d1cecf)

However this does clog up the traces:

![image](https://github.com/user-attachments/assets/76933635-0b5b-4f9e-a31b-66bc11ccf701)

They can be filtered out like so:

![image](https://github.com/user-attachments/assets/1281f5e8-be51-41a5-a6e6-aeb64df7a50b)

But the filters would need to be added every restart of Aspire.

Perhaps a "prefilled" filter button to filter out health checks (which we are aware of because of `.WithHttpsHealthCheck("/health")` etc.
